### PR TITLE
ADD: Added chat feature.

### PR DESCRIPTION
**Pull request by zoef-yt**
In this PR, I added chat functionality and fixed the comment section.

### CODE REVIEW

**Review by tonymike99 — APPROVED** [review]
Hey Zoef,
Super cool stuff! Nice code. Good to merge.

**Review by sudipkundu999 — APPROVED** [review]
The code looks good and the functionality works as expected.
Good work on the chat feature 🚀

File: src/Components/SinglePostCard/AllPostComments.jsx [hunk 1]
```diff
@@ -12,10 +12,12 @@ const AllPostComments = ({ closeComments, commentSection, photoURL, displayName,
...
 		if (authUser) {
-			if (userComment) {
-				await addCommentsToPost(postId, userId, userComment);
-				setUserComment('');
-				setTriggerEffect(!triggerEffect);
+			if (userComment.trim() !== '') {
+				if (userComment) {
+					await addCommentsToPost(postId, userId, userComment);
+					setUserComment('');
+					setTriggerEffect(!triggerEffect);
```
Comment (muskanbagrecha) [inline src/Components/SinglePostCard/AllPostComments.jsx:19]: This is not a pure update.

**Review by muskanbagrecha — APPROVED** [review]
Code looks very clean. CSS is nicely written. Love the chat functionality. Great UI. Good job.

**Pull request merged**